hack/make-rules/test-cmd-util.sh

```diff
@@ -2574,6 +2575,22 @@ run_rs_tests() {
   fi
 }
 
+run_daemonset_tests() {
```

Comment:
I'm just wondering if this test adds anything over the unit test you've already put in.